packages/Overlay/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.0.15
+
+### Patch Changes
+
+- add option to disable the click events on backdrop
+
```

Comment:
Could you revert this file and use `yarn changeset` command? https://github.com/acl-services/paprika/wiki/Releases#what-contributors-have-to-do

Reply:
`yarn changeset` doesn't seem to work on overlay for some reason. I got this error.. should I do `yarn changeset in it` instead? 

<img width="898" alt="Screen Shot 2022-03-14 at 12 06 57 PM" src="https://user-images.githubusercontent.com/92405042/158252858-cbece3f4-133c-424d-9222-affc338709a0.png">